0000_10_cluster-kube-apiserver-operator_02_cabundleconfigmap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-core-operators
+  name: service-ca-bundle-configmap
+  annotations:
+    service.alpha.openshift.io/inject-cabundle: "true"
```

Comment:
clever, but don't do this.  @openshift/sig-security and @mrogers950  in particular should be able to tell you which configmap in their operator namespace provides *just* the ca bundle for their signer.  For compatibility, this one will be injected with their signer *and* the signer for the old way.  We want to keep them separate.

Reply:
@juanvallejo  it's the `signing-cabundle` configmap in `openshift-service-cert-signer`. The data item is `cabundle.crt`and contains the PEM bundle.